composer.json

```diff
@@ -9,7 +9,9 @@
     }
   ],
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "spryker/checkout-extension": "^1.0.0",
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose Yes now